charts/openfaas-cloud/templates/ingress-wildcard.yml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: openfaas-ingress
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    nginx.ingress.kubernetes.io/limit-connections: "20"
+    nginx.ingress.kubernetes.io/limit-rpm: "600"
+  labels:
+    app: faas-netesd
+spec:
+  {{ if .Values.tls.enabled }}
+  tls:
+  - hosts:
+    - {{ printf "*.%s" .Values.global.rootDomain | quote }}
+    secretName: wildcard-{{ .Values.global.rootDomain }}-cert
+  {{ end }}
+  rules:
+  - host: {{ printf "*.%s" .Values.global.rootDomain | quote }}
+    http:
+      paths:
+      - path: /
+        backend:
+          serviceName: edge-router
+          servicePort: 8080
+  - host: {{ printf "gateway.%s" .Values.global.rootDomain | quote }}
```

Comment:
Please can we drop gateway both here and in orc-bootstrap? It isn't required and could expose some functions we want to keep private, Linkerd also doesn't like it when doing the host mapping.

Reply:
That'll leave us with -> IngressController -> Edge Router -> GW